src/engraving/rendering/score/slurtielayout.cpp

```diff
@@ -1018,7 +1010,7 @@ void SlurTieLayout::avoidCollisions(SlurSegment* slurSeg, PointF& pp1, PointF& p
             }
         } else if (!slurSeg->isEndPointsEdited()) {
             // In the odd iterations, adjust the end points
-            // Slurs steeper than 45° are gently compensated
+            // Slurs steeper than 45Ãƒâ€šÃ‚Â° are gently compensated
```

Comment:
Wonder what happened here (was it Uncrustify? 😄)

Reply:
Urghhhhhhhhhh yes